db/migrate/20161212205845_drop_follows.rb

```diff
@@ -0,0 +1,5 @@
+class DropFollows < ActiveRecord::Migration
```

Comment:
This is quite weird on sufia's part. They are actively editing their migrations, instead of creating new migrations? Seems to be an upgrade nightmare?

Like they originally had this migration but in 7.2 they just dropped all the migrations for the follows table:
https://github.com/projecthydra/sufia/commit/b60de8b4bc7bd081f818ee498535d06ca77f2ac3

Lots of other examples in the migration commit history too.

Regardless follows table isn't needed anymore. So I wrote my own drop migration below.